### DISCUSSION TITLE
Just a question about future support...

### DISCUSSION THREAD

**Discussion by shinyangdevelop** [Textualize/textual#1046] [Q&A] [answered]
Is there any plan for making Textual in node.js package, or in go? Because there are lots if programmers who are using javascript or go, and Golang is easy to compile and distribute.

**Comment by davep** [comment]
Ultimately @willmcgugan is the one to give the definitive answer, but Textual is a project that is building a Python TUI framework. The other languages you mention seem fairly well served by existing packages.

**Comment by willmcgugan** [comment, marked as answer]
It's not out of the question. One day perhaps. But not until the Python version is mature. If you want a TUI in go, check out charm.sh
